src/app/onboarding.tsx

```diff
@@ -35,7 +35,7 @@ export default function Onboarding() {
       </View>
       <SafeAreaView className="mt-6">
         <Button
-          label="Let's Get Started "
+          label="Let's Get Started"
```

Comment:
I know this text was here before this PR, but we can move it to the translations file, what do you think?

Reply:
Done!